guardian/support-dotcom-components
# Ticker update to banner and epics

## What does this change?
Update to the ticker design to keep in line with what is current in [System Design ](https://www.figma.com/design/mkyzlXSnCGEOIRNncwziDC/Progress-bar-test?node-id=1-821&t=2pbFhkB0l1gZcf5D-0). This changes what the ticker looks like but the customisation of the copy and colours is still enabled.

There is a new storybook file to show the new ticker with three scenarios people ticker, money ticker and what happens when the goal is met.
  
## How to test
On Storybook open the Ticker component storybook and/or check out the ticker on the Designable Banner stories and/or the MomentTemplateBanner


## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
